rq_dashboard/web.py

```diff
@@ -277,7 +289,9 @@ def serialize_queue_names(worker):
         dict(
             name=worker.name,
             queues=serialize_queue_names(worker),
-            state=str(worker.get_state())
+            state=str(worker.get_state()),
+            current_job_tooltip=serialize_current_job_tooltip(
+                worker.get_current_job()),
```

Comment:
I would have prefer machine-readable version with something like this
```javascript
"current_job": {
    "id": "xxxxxx",
    "description": "job description",
    "created_at": "2018-01-01 20:00:11" 
}
```
to be returned from backend and then formatted on frontend counterpart to be displayed as a tooltip. This way this info can be used as API data and formatting can ba adjusted without changing of API response format.

Is there a reason why you have decided to return It as a formatted string?

And the idea of tooltip looks cool, I was always missing this feature myself.

Reply:
Hey. JSON format is definitely more suitable, the reason was mostly because I wanted to present that to non-technical users in my use case. It works quite nice by the way.

Do you have an idea where to display this information better, rather than non-intuitive hover on the icon?